installation/supported-platforms.md

```diff
@@ -14,7 +14,7 @@ The following operating systems and architectures are supported in Fluent Bit.
 |  | [Debian 12 \(Bookworm\)](linux/debian.md) | x86\_64, Arm64v8 |
 |  | [Debian 11 \(Bullseye\)](linux/debian.md) | x86\_64, Arm64v8 |
 |  | [Debian 10 \(Buster\)](linux/debian.md) | x86\_64, Arm64v8 |
-|  | [Ubuntu 22.04 \(Jammy Jellyfish\)](linux/ubuntu.md) | x86\_64, Arm64v8 |
+|  | [Ubuntu 22.04 \(Jammy Jellyfish\)](linux/ubuntu.md) | x86\_64, Arm64v8, s390x |
```

Comment:
Is this true? We do not seem to explicitly do anything to provide any DEBs for s390x which is what this table is showing - actual packages are built and provided officially from our repos. Not that it can be but that it is.

The only changes in the linked PR are to build images, not packages, so want to confirm. If we want to build it via the normal `./packaging/build.sh -d <distro>` then how do we do that?

I would extend the statement about container images being provided with something along the lines of "it is possible to build for Ubuntu 22.04 as well".

Reply:
removed..